themes/default/assets/sass/_accordion.scss

```diff
@@ -13,6 +13,10 @@
         transition: max-height 1s ease-out;
     }
 
+    .accordion-item-body-no-animation {
```

Comment:
TBH, I much prefer the behavior _without_ the animation, as it actually felt pretty sluggish to me. This feels snappy and does what I expect, so I'd personally suggest we just nix the animation altogether.

Reply:
👍 Yeah I agree with you. I'll cut a different PR to update the other accordions once this makes it through.